.github/workflows/release.yaml

```diff
@@ -0,0 +1,17 @@
+name: Release
+
+on:
+  push:
+    tags: ["v*"]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Login to GitHub Container Registry
+      run: echo "$GITHUB_TOKEN" | docker login ghcr.io --username "$GITHUB_ACTOR" --password-stdin
```

Comment:
Is it safer to use https://github.com/docker/login-action#github-container-registry

Reply:
I believe that using action just to do this is just making it harder to understand what is being done inside, but I don't have a strong opinion, so I will change it.

Reply:
I've updated.